nextmv/nextmv/cloud/application.py

```diff
@@ -3846,39 +3859,6 @@ def __input_set_for_scenario(self, scenario: Scenario, scenario_id: str) -> Inpu
 
         raise ValueError(f"Unknown scenario input type: {scenario.scenario_input.scenario_input_type}")
 
-    def __validate_dir_path_and_configuration(
```

Comment:
This function is important, as mentioned, for validating that if we are submitting csv-archive, or multi-file, then the input_dir_path must be provided. In the same way, if we specify that arg, we must be working with csv or multi-file.

Reply:
I can bring it back, but it will be quite more lean then. I suppose users may still want to provide the input type on run config, but it's not recommended anymore basically. :blush: 

I'll look into finding a middle ground.